docs/aws : incite to delete the config after 1st boot (#576)
coreos/fedora-coreos-docs: modules/ROOT/pages/provisioning-aws.adoc

ssh core@<ip address>
----

> == Remote ignition configuration

Reviewer: ```suggestion
== Remote Ignition configuration
```

      source: s3://instance1-infra/bootstrap.ign
----

Then you can launch the instance using the same command as xref:#_customized_example[], passing the minimal configuration you just created.

Once the first boot is completed, make sure to delete the configuration as it may contain sensitive data.
See xref:#_configuration_cleanup[].

== Configuration cleanup

> If you need to have secrets in your ignition configuration you should store it into a S3 bucket and have a minimal configuration in user-data.

Reviewer: ```suggestion
If you need to have secrets in your Ignition configuration you should store it into a S3 bucket and have a minimal configuration in user-data.
```

Once the instance has completed the first boot, clear the S3 bucket as any process or container running on the instance could access it.

> See the https://coreos.github.io/ignition/operator-notes/#secrets[ignition documentation] for more advice on secret management.

Reviewer: ```suggestion
See the https://coreos.github.io/ignition/operator-notes/#secrets[Ignition documentation] for more advice on secret management.
```

      source: s3://instance1-infra/bootstrap.ign
----

Then you can launch the instance using the same command as xref:#_customized_example[], passing the minimal configuration you just created.

Once the first boot is completed, make sure to delete the configuration as it may contain sensitive data.
See xref:#_configuration_cleanup[].

== Configuration cleanup

Once the instance have completed the first boot, we recommend cleaning up the configuration files.

> Any container running on the instance could be able to read the config, raising security concerns.

Reviewer: Let's link to https://coreos.github.io/ignition/operator-notes/#secrets

      source: s3://instance1-infra/bootstrap.ign
----

Then you can launch the instance using the same command as xref:#_customized_example[], passing the minimal configuration you just created.

Once the first boot is completed, make sure to delete the configuration as it may contain sensitive data.
See xref:#_configuration_cleanup[].

== Configuration cleanup

> Once the instance have completed the first boot, we recommend cleaning up the configuration files.
> Any container running on the instance could be able to read the config, raising security concerns.

Reviewer: ```suggestion
Once the instance has completed the first boot,we recommend removing the Ignition config from the S3 bucket as any process or container running on the instance could access it.
```

[source,bash]
.Deleting the Ignition configuration from the s3 bucket
----
NAME='instance1'
aws s3 rm CONFIG s3://$NAME-infra/bootstrap.ign
----

Optionnally, you can delete the whole bucket:
[source,bash]
.Deleting the s3 bucket
----
NAME='instance1'
aws s3 rb s3://$NAME-infra
----

NOTE: The instance's user data cannot be modified without stopping the instance.
If you need to have secrets in your ignition configuration you should store it into a s3 bucket and have a minimal configuration in user-data.
Make sure to clear the s3 bucket when the first boot is completed.

> See the https://coreos.github.io/ignition/operator-notes/#secrets[ignition documentation] for more advice on secret management.

Reviewer: Let's move that into a single paragraph with the rest at the top of this section.
